src/tools/rest-backend/README.md

```diff
@@ -56,6 +56,31 @@ The REST service can also be configured. The file `rest-backend-config.js` in th
 
 To stop the service, run `sh stop-rest-backend.sh` in the same path.
 
+### Configure as service
+
+To configure the rest-backend as service, it is possible to use `systemd` on most systems.
+
+1) Create a new service file under `/etc/systemd/system`, e.g. `touch /etc/systemd/system/rest-backend.service`.
```

Comment:
The folder should be existing already if `systemd` is installed (quite sure this won't be the first service). But even if this is not the case, I guess the user will know what to do when getting an error because of a non-existing directory.

I changed it to your suggestion; didn't know `cat` supports heredoc.